docs/millennium-os/manual/chapters/10_introduction.md

```diff
@@ -55,33 +55,33 @@ MillenniumOS uses the concept of features to enable and disable functionality. S
 
 If you don't have the hardware for one or more of these features or you disable the feature, MillenniumOS will fall back on other, manual methods to achieve the same goal.
 
-This allows us to use the same gcode dialect for machines which have any combination of feature support.
+This allows us to use the same gcode dialect for machines that have any combination of feature support.
 
-This may be hard to visualise, so here are some examples:
+This may be hard to visualize, so here are some examples:
 
 ### Toolsetter and Touch Probe Disabled
 
-You will be guided through manual work-piece probing when the machine needs to know the origin of a work-piece. Every time a tool is changed, you will be guided through a manual process to re-zero the origin in the Z axis, as the tool length will have changed.
+You will be guided through manual work-piece probing when the machine needs to know the origin of a work-piece. Every time a tool is changed, you will be guided through a manual process to re-zero the origin in the Z-Axis, as the tool length will have changed.
 
-You will be able to do everything, but it will take a while as this will involve manual probing at every step.
+You can do everything, but it will take a while as this will involve manual probing at every step.
```

Comment:
```suggestion
You can do everything required for job set up via this process, but it will take a long time as each step will involve manual probing methods.
```